### PR TITLE
docs(ui5-input): add storybook samples

### PR DESCRIPTION
**Pull request by Okiana**
Add storybook sample for ui5-input

### CODE REVIEW

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 1]
```diff
@@ -66,344 +98,190 @@ export const Template1: StoryFn = () => html`
...
-			}
-			async function loadList() {
-				var response = await fetch("../assets/data/products.json");
-				var products = await response.json();
-				var query = dialogSearchInput.value.toLowerCase();
-				itemsList.innerHTML = "";
-				products
+	</ui5-dialog>
+	<script>
+		valueHelpInput.addEventListener("input", loadSuggestions);
```
Comment (elenastoyanovaa) [inline packages/playground/_stories/main/Input/Input.stories.ts:198]: where are those variables defined?

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
- Change the PR name to docs
- Remove the ts-ignore
- The formatting of the code (in the storybook when you click on the show code button) is wrong - wrong tabulation or several items on concatenated on one line, which makes the code very unreadable.

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 2]
```diff
@@ -24,30 +32,54 @@ export default {
...
-		<ui5-input class="samples-margin samples-responsive-margin-bottom" disabled="" value="Disabled Input"></ui5-input>
-	</div>
-`;
-
+export const BasicInput = Template.bind({});
+BasicInput.args = {
+	showClearIcon: true,
+	value: "Input"
+};
+BasicInput.storyName = "Basic Input";
```
[the remark this answers is not in the document]
Comment (Okiana) [reply packages/playground/_stories/main/Input/Input.stories.ts:64]: Done

File: packages/playground/_stories/main/Input/Input.stories.ts (hunk 1 above)
Comment (Okiana) [reply packages/playground/_stories/main/Input/Input.stories.ts:198]: Done

**Comment by Okiana** [comment]
> * Change the PR name to docs
> * Remove the ts-ignore
> * The formatting of the code (in the storybook when you click on the show code button) is wrong - wrong tabulation or several items on concatenated on one line, which makes the code very unreadable.

Done

**Review by MapTo0 — CHANGES_REQUESTED** [review]
- do not mix tabs and spaces in the file. format with tabs only
- Could you please rewrite the examples with arrow functions instead of fn declarations

The output in the playground looks strange:

image

Some private props are there but should not be part of the example. Do we have a solution for that one? @dobrinyonkov 

I would say that we remove search field example since we have the value help example which basically sets the same slot - `icon` with and presents an input with a custom icon

Input with label example is not working correctly - clicking on the label does not focus the input, also I guess the ACC is broken @elenastoyanovaa 

Consider also combining or removing one of the value states examples. I would suggest that we have just 1 custom implemented value state message since changing the value-state property is quite straight forward.

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 3]
```diff
@@ -24,386 +31,266 @@ export default {
...
-					li.text = item;
-					input.appendChild(li);
-				});
+export const InputSuggestions = Template.bind({});
+InputSuggestions.decorators = [
+	(story) => {
+		return html`
+		${story()}
+<script>
+	let value,
```
[the remark this answers is not in the document]
Comment (Okiana) [reply packages/playground/_stories/main/Input/Input.stories.ts:70]: Done

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 4]
```diff
@@ -24,386 +31,266 @@ export default {
...
+		li,
+		input = document.getElementById("input-${index}"),
+		ui5_database_entries = ["Argentina", "Albania", "Algeria", "Angola",
+		"Austria",  "Australia", "Bulgaria", "Canada", "Columbia", "Croatia", "Denmark",
+		"England", "Finland", "France", "Germany", "Hungary", "Ireland", "Italy", "Kuwait",
+		"Luxembourg", "Mexico", "Morocco", "Norway", "Paraguay", "Philippines", "Portugal",
+		"Spain", "Sweden", "Sri Lanka", "Senegal", "United Kingdom", "USA" ];
+	input.addEventListener("input", function(event) {
```
[the remark this answers is not in the document]
Comment (Okiana) [reply packages/playground/_stories/main/Input/Input.stories.ts:78]: Done

**Comment by Okiana** [comment]
> image
> Some private props are there but should not be part of the example. Do we have a solution for that one? @dobrinyonkov ->related to: #7035 


All other comments are done.

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 5]
```diff
@@ -7,15 +7,22 @@ import type { UI5StoryArgs } from "../../../types.js";
...
 
 const component = "ui5-input";
+let index = 0;
 
 export default {
     title: "Main/Input",
     component,
-    subcomponents: {'SuggestionItem' : 'ui5-suggestion-item', 'SuggestionGroupItem' : 'ui5-suggestion-group-item'},
+    subcomponents: {
+		SuggestionItem: 'ui5-suggestion-item',
```
Comment (MapTo0) [inline packages/playground/_stories/main/Input/Input.stories.ts:23]: fix tabulation, do not mix spaces and tabs

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 6]
```diff
@@ -7,403 +7,260 @@ import type { UI5StoryArgs } from "../../../types.js";
...
-	</div>
-`;
-Template1.parameters = {
-	docs: {
-		story: {
-			// Opt-out of inline rendering
-			inline: false,
-		},
+		}
+		[].slice.call(input${index}.children).forEach((child) => {
```
Comment (MapTo0) [inline packages/playground/_stories/main/Input/Input.stories.ts:85]: Use `Array.from` ?

**Review by MapTo0 — APPROVED** [review]

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**